lib/utils/azure_queues.ts

```diff
@@ -163,8 +191,8 @@ export async function handleQueueProcessingFailure(
     const shouldTriggerARetry = await updateMessageVisibilityTimeout(
       queueService,
       queueName,
-      queueMessage
-    );
+      queueMessageBindings
+    ).catch(winston.error);
```

Comment:
If i remember well exceptions here should bubble up to the caller which in turns decides to retry the action. You can verify this looking at the caller level into the queue processing functions. In case, get rid of the catch here.